### PR TITLE
WW-91 Provide entryPointWikiId var only when exists

### PR DESCRIPTION
**Pull request by dianafa**
## Links

* https://wikia-inc.atlassian.net/browse/WW-91

## Description

Use this var only when exists

## Reviewers

@Wikia/west-wing @bkoval @rafalkalinski 

ask for var only when avaiable

### CODE REVIEW

**Comment by SebastianMarzjan** [comment]
```
{{#if LGTSocial}}
  LGTM
{[/if}}
```

File: front/auth/views/_layouts/auth.hbs [hunk 1]
```diff
@@ -85,8 +85,11 @@
 					user: 'en',
 				},
 			});
-			// place where we can check wikiId of the entrypoint
-			M.provide('entryPointWikiId', {{entryPointWikiId}});
+
+			// wikiId of the entrypoint if available
+			{{#if entryPointWikiId}}
```
Comment (bkoval) [inline front/auth/views/_layouts/auth.hbs]: Shouldn't we have a default value for it?

Comment (dianafa) [inline front/auth/views/_layouts/auth.hbs]: we don't need such for now. If value is not set in the place we need it:
- means something is terribly wrong and we need to fix it anyways
- default value will not bring us any benefit/more information here - we're fine with undefined

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**